Sources/SuperwallKit/Models/Triggers/Experiment.swift

```diff
@@ -10,14 +10,18 @@ import Foundation
 
 /// A campaign experiment that was assigned to a user.
 ///
-/// An experiment is part of a [Campaign Rule](https://docs.superwall.com/docs/campaign-rules) defined in the Superwall dashboard. When a rule is matched, the user is assigned to an experiment, which  is a set of paywall variants determined by probabilities. An experiment will result in a user seeing a paywall unless they are in a holdout group.
+/// An experiment is part of a [Campaign Rule](https://docs.superwall.com/docs/campaign-rules)
+/// defined in the Superwall dashboard. When a rule is matched, the user is
+/// assigned to an experiment, which is a set of paywall variants determined
+/// by probabilities. An experiment will result in a user seeing a paywall unless
+/// they are in a holdout group.
 ///
 /// To learn more, read <doc:Ecosystem>.
-public struct Experiment: Equatable, Hashable, Codable {
+public struct Experiment: Equatable, Hashable, Codable, Sendable {
```

Comment:
I think we might want to expose this in objc

Reply:
Done!

Reply:
Awesome :D Could you take a quick review of this class – there are a few things here that won't be accessible in objc including the variant, variant type, etc.

Also pref for something like `@objc(SWKExperiment)`

Reply:
Done!